ocrd_monitor/ocrdmonitor/server/templates/base.html.j2

```diff
@@ -1,65 +1,65 @@
-          </a>
-          <div class="navbar-item has-dropdown is-hoverable">
-            <a class="navbar-link">
-              More
-            </a>
-            <div class="navbar-dropdown">
-              <a href="https://ocr-d.de/" class="navbar-item">
-                OCR-D
-              </a>
-              <hr class="navbar-divider">
-              <a href="https://github.com/slub/ocrd_manager/issues" class="navbar-item">
-                Report an issue
-              </a>
-            </div>
-          </div>
-        </div>
-      </div>
-    </nav>
-  <section class="section">
-    <div id="main" class="container">
-      <h1 class="title">
-        {% block headline %}{% endblock %}
-      </h1>
-      {% for message in get_flashed_messages() %}
-      <div class="flash">{{ message }}</div>
-      {% endfor %}
-      {% block content %}{% endblock %}
-    </div>
-  </section>
-  </body>
-</html>
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% block meta %}{% endblock %}
+    <title>{% block title %}{% endblock %} - OCR-D Monitor</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
+    <link rel="stylesheet" href="{{ url_for('static', path='style.css') }}">
+  </head>
+  <body>
+    <nav class="navbar" role="navigation" aria-label="main navigation">
+      <div class="navbar-brand">
+        <a class="navbar-item" href="/">
+          <b>OCRD-Monitor</b>
+        </a>
+    
+        <a role="button" class="navbar-burger" aria-label="menu" aria-expanded="false" data-target="navbarBasicExample">
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+        </a>
+      </div>
+    
+      <div id="navbarBasicExample" class="navbar-menu is-active">
+        <div class="navbar-start">
+          <a href="{{ url_for('workspaces.list') }}" class="navbar-item">
+            Workspaces
+          </a>
+          <a href="{{ url_for('jobs') }}" class="navbar-item">
+            Jobs
+          </a>
+          <a href="#" class="navbar-item">
+            Logs
+          </a>
+          <div class="navbar-item has-dropdown is-hoverable">
+            <a class="navbar-link">
+              More
+            </a>
+            <div class="navbar-dropdown">
+              <a href="https://ocr-d.de/" class="navbar-item">
+                OCR-D
+              </a>
+              <hr class="navbar-divider">
+              <a href="https://github.com/slub/ocrd_manager/issues" class="navbar-item">
+                Report an issue
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+  <section class="section">
+    <div class="container">
+      <h1 class="title">
+        {% block headline %}{% endblock %}
+      </h1>
+      {# {% for message in get_flashed_messages() %}
```

Comment:
Should be removed if not needed.

Reply:
FastAPI, doesn't have an equivalent to flashed messages, I'll come up with a replacement.

Reply:
I have implemented an error message that shows up when the browser can't connect to broadway or loses its connection.
It's not a general error-displaying mechanism as in flask yet, but it should be sufficient for the functionality we have so far.